library/psa_crypto.c

```diff
@@ -5488,10 +5531,10 @@ static psa_status_t psa_key_derivation_input_internal(
         goto exit;
 
 #if defined(MBEDTLS_PSA_BUILTIN_ALG_HKDF)
-    if( PSA_ALG_IS_HKDF( kdf_alg ) )
+    if( PSA_ALG_IS_HKDF( kdf_alg ) || PSA_ALG_IS_HKDF_EXTRACT( kdf_alg ) ||
```

Comment:
Four places use the same set of conditions - maybe consider refactoring? Minor suggestion.

Reply:
Fixed